src/TrixiParticles.jl

```diff
@@ -21,16 +20,16 @@ using SciMLBase: CallbackSet, DiscreteCallback, DynamicalODEProblem, u_modified!
 @reexport using StaticArrays: SVector
 using StaticArrays: @SMatrix, SMatrix, setindex
 using StrideArrays: PtrArray, StaticInt
-using ThreadingUtilities
 using TimerOutputs: TimerOutput, TimerOutputs, print_timer, reset_timer!
 using TrixiBase: trixi_include
+@reexport using PointNeighbors: TrivialNeighborhoodSearch, GridNeighborhoodSearch
+using PointNeighbors: PointNeighbors, for_particle_neighbor
```

Comment:
Isn't this redundant `PointNeighbors: PointNeighbors`?

Reply:
No. This allows you to use `PointNeighbors.xyz` inside TrixiP for some unexported symbol `xyz`.